-----------------------------------------------------------------
Revision: 41389df89a4a6a007d41cec33e33043cd41ea159
Ancestor: 326d6e6c8c6a955cdba84f1f802fc21ec3e6b662
Author: datallah@pidgin.im
Date: 2007-09-17T16:24:24
Branch: im.pidgin.pidgin

Modified files:
        libpurple/protocols/jabber/message.c

ChangeLog: 

more leak fixes.

============================================================
--- libpurple/protocols/jabber/message.c	2484754364477f8cacff4778fa87396b81924175
+++ libpurple/protocols/jabber/message.c	3bc876da8fe16d41770af8d980dce8457b3ac61a
@@ -41,7 +41,10 @@ void jabber_message_free(JabberMessage *
 	g_free(jm->body);
 	g_free(jm->xhtml);
 	g_free(jm->password);
+	g_free(jm->error);
+	g_free(jm->thread_id);
 	g_list_free(jm->etc);
+	g_list_free(jm->eventitems);
 
 	g_free(jm);
 }
